lib/node_modules/@stdlib/math/base/ops/lib/index.js

```diff
@@ -72,15 +72,6 @@ setReadOnly( ns, 'cadd', require( '@stdlib/complex/float64/base/add' ) );
 */
 setReadOnly( ns, 'cmul', require( '@stdlib/complex/float64/base/mul' ) );
 
-/**
-* @name add5
-* @memberof ns
-* @readonly
-* @type {Function}
-* @see {@link module:@stdlib/math/base/ops/add5}
-*/
-setReadOnly( ns, 'add5', require( '@stdlib/math/base/ops/add5' ) );
```

Comment:
@gururaj1512 This is fine for this PR; however, this change needs to happen during the package removal step. This is actually a breaking change, so should not be included in this commit.

Reply:
For reference, see our Git commit guide: https://github.com/stdlib-js/stdlib/tree/develop/docs/style-guides/git

Reply:
@gururaj1512 Actually, sorry. Disregard my comment above. Instead of removing, just update the path. We'll clean up the namespace once everything is moved.